pages/api/iiif/metadata/[label].ts

```diff
@@ -49,10 +50,13 @@ export default function handler(req, res) {
                 const representative =
                   term.values[Math.floor(Math.random() * term.values.length)]
                     .manifestId;
+                const thumbnail = term.values[Math.floor(Math.random() * term.values.length)].thumbnail;
                 return {
                   label: term.value,
-                  summary: `${term.values.length} Items`,
-                  id: representative,
+                  summary: `View objects with a label of "${grouped.label}" and value of "${term.value}".`,
+                  id: `${origin}/api/iiif/metadata/${slugify(grouped.label)}/${slugify(term.value)}`,
```

Comment:
```suggestion
                  id: representative,
```

Bloom expects this to be a real manifest and we can just pass it `representative` which is is a stand-in for the original source manifest.

Reply:
@mathewjordan are you saying that for now we should just point at the representative manifest.  i think this is fine, but ulitmately that id should point at a collection manifest like `https://can-git-collectionthumbnail-mathewjordan.vercel.app/api/iiif/metadata/Artist/Braddock-Paige` which would have an `items` property that included the things that would be rendered on the corresponding homepage.